Makefile

```diff
@@ -24,8 +24,8 @@ export GOFLAGS GO_LDFLAGS CGO_ENABLED
 
 prepare: install-tools go-vendor
 
-test:
-	go test -v -cover -coverprofile=$(COVERAGEOUT) $(shell go list ./... | grep -v integration)
+test: prepare
```

Comment:
make install-tools could be the pre-req here or that could even be added as a build step. I just added `prepare` for now to get the build passing